typescript/phoenix-sdk/tests/openOrders.ts

```diff
@@ -0,0 +1,137 @@
+import { Connection } from "@solana/web3.js";
+
+import * as Phoenix from "../src";
+
+const getOrderSequenceNumber = (orderId: Phoenix.OrderId): bigint => {
+  const num = BigInt(orderId.orderSequenceNumber.toString()); // 64-bit integer (maximum value)
+  const low = Number(num & BigInt(0xffffffff)); // lower 32 bits
+  const high = Number(num >> 32n); // upper 32 bits
+  const inverseLow = ~low >>> 0; // perform bitwise NOT on lower 32 bits
+  const inverseHigh = ~high >>> 0; // perform bitwise NOT on upper 32 bits
+  const sequenceNumber = (BigInt(inverseHigh) << 32n) | BigInt(inverseLow); // combine the results
+  return sequenceNumber;
+};
+
+export async function watch() {
+  const connection = new Connection("http://127.0.0.1:8899");
+  const phoenix = await Phoenix.Client.create(connection, "localhost");
+
+  const market = Array.from(phoenix.markets.values()).find(
+    (market) => market.name === "SOL/USDC"
+  );
+  if (!market) throw new Error("Market not found");
+
+  let traderKey;
+  for (const [trader, traderState] of market.data.traders) {
```

Comment:
this is sample code that people will read. add comment that this is finding the first trader that has something on the book